apps/payments/next/app/[locale]/[offeringId]/[interval]/checkout/[cartId]/layout.tsx

```diff
@@ -99,6 +99,7 @@ export default async function CheckoutLayout({
                 interval={cart.interval}
               />
             }
+            locale={ l10n.locale }
```

Comment:
Is there a reason this uses `l10n.locale` or could this use the locale from params instead?